ceph_iscsi_config/settings.py

```diff
@@ -94,6 +94,7 @@ def normalize(k, v):
         return v
 
     defaults = {"cluster_name": "ceph",
+                "cluster_client_name": "client.admin",
                 "gateway_keyring": "ceph.client.admin.keyring",
```

Comment:
Nit: I wonder if this should default to ```None``` / blank now so that librados can fallback to its default search paths / naming conventions for the keyring (i.e. ```<cluster-name>.<cephx name>.keyring```). Or I wonder if it just makes sense to drop the keyring since it can be specified via the Ceph configuration file anyway.

Reply:
I’m in favor of dropping it. How should we handle the upgrade? Log a warning saying that “gateway_keyring” is ignored if the Ceph-iscsi conf file still has the option set?

Reply:
I doubt anyone has ever used that override, so it should be safe to just remove.